acceptancetests/assess_upgrade_series.py

```diff
@@ -78,9 +78,7 @@ def reboot_machine(client, machine):
     try:
         client.juju('ssh', (machine, 'sudo shutdown -r now'))
     except subprocess.CalledProcessError as e:
-        if e.returncode != 255:
-            raise e
-        log.info("Ignoring `juju ssh` exit status after triggering reboot")
+        log.info("Ignoring `juju ssh` exit code {} after triggering reboot".format(e.returncode))
```

Comment:
As you can see, 255 is not surfaced in the caught exception. It's 1. Honestly, it's not worth playing with in these Python-based tests. If there's some other failure issuing a reboot, particularly after `do-release-upgrade` the test will fail anyway.

Reply:
Interesting. IIRC I added this check specifically because of the 255 status. But in any case, I agree it's best to ignore it altogether. There may be more cases of this particular check in the tests which we should probably also change to be consistent.